sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -910,6 +929,13 @@ mod checked {
                 written_objects.insert(id, object);
             }
 
+            for package in new_packages {
```

Comment:
This got moved down as we need to use `new_package` up above when loading the type. This shouldn't have any effect, but double check me 🙏🏻